File: common-java/src/test/java/net/yamac/springboot2/example/java/ExampleServiceTest.java
package net.yamac.springboot2.example.java;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

import static org.assertj.core.api.Assertions.assertThat;

@RunWith(SpringRunner.class)
@SpringBootTest
public class ExampleServiceTest {

    @Autowired
    private ExampleService exampleService;

    @Test
    public void testGetMessageReturnsCorrectValue() {
        assertThat(exampleService.getMessage()).isNotNull();
        assertThat(exampleService.getMessage()).contains("Hello World Java").contains("Test");
    }
}

File: common-kotlin/src/test/kotlin/net/yamac/springboot2/example/kotlin/ExampleServiceTest.kt
package net.yamac.springboot2.example.kotlin

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.junit4.SpringRunner

@RunWith(SpringRunner::class)
@SpringBootTest
class ExampleServiceTest {

    @Autowired
    private val exampleService: ExampleService? = null

    @Test
    fun testGetMessageReturnsCorrectValue() {
        assertThat(exampleService!!.getMessage()).isNotNull()
        assertThat(exampleService.getMessage()).contains("Hello World Kotlin").contains("Test")
    }
}
